src/ImageSearchModal.jsx

```diff
@@ -110,7 +110,7 @@ export class ImageSearchModal extends React.Component {
                                 results = results.concat(JSON.parse(result.value));
                             } else {
                                 dialogError = _("Failed to search for new images");
-                                dialogErrorDetail = cockpit.format(_("Failed to search for images: $0"), result.reason);
+                                dialogErrorDetail = result.reason ? cockpit.format(_("Failed to search for images: $0"), result.reason.message) : _("Failed to search for images.");
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1172-20230106-100359-1eb6bb49-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)